cmd/kops/get.go

```diff
@@ -46,9 +46,15 @@ var (
 	# Get a cluster
 	kops get cluster k8s-cluster.example.com
 
-	# Get a cluster YAML cluster spec
+	# Get a cluster YAML cluster desired configuration
 	kops get cluster k8s-cluster.example.com -o yaml
 
+	# Get a cluster YAML cluster actual configuration
+	kops get cluster k8s-cluster.example.com --full -o yaml
```

Comment:
I would not put this here, as we do not want users to use full yaml, or put a big warning here